examples/users_exmaple.pp

```diff
@@ -0,0 +1,8 @@
+registry_key { "HKU\\${windows_currentuser_sid}\\Test2":
+  ensure => present
+}->

```

Comment:
This example shows the user as part of the key path, but below you've added the `user` as a new parameter to the registry_key.